config/systems-uenv/clariden-uenv.py

```diff
@@ -0,0 +1,194 @@
+    },
+    {
+        'name': f'amdgpu',
+        'scheduler': 'slurm',
+        'time_limit': '10m',
+        'environs': environ_names,
+        'max_jobs': 100,
+        'extras': {
+            'cn_memory': 500,
+        },
+        'access': ['-pamdgpu'] + uenv_access,
+        'resources': [
+            {
+                'name': 'switches',
+                'options': ['--switches={num_switches}']
+            },
+            {
+                'name': 'memory',
+                'options': ['--mem={mem_per_node}']
+            },
+        ],
+        'features': ['gpu', 'amdgpu', 'remote', 'uenv'],
+        'launcher': 'srun'
+    },
+    {
+        'name': f'cpu',
+        'scheduler': 'slurm',
+        'time_limit': '10m',
+        'environs': environ_names,
+        'max_jobs': 100,
+        'extras': {
+            'cn_memory': 500,
+        },
+        'access': ['-pcpu'] + uenv_access ,
+        'resources': [
+            {
+                'name': 'switches',
+                'options': ['--switches={num_switches}']
+            },
+            {
+                'name': 'memory',
+                'options': ['--mem={mem_per_node}']
+            },
+        ],
+        'features': ['remote', 'uenv'],
+        'launcher': 'srun'
+    }
+]
+
+if image_environments:
+    actual_environs = []
+
+for k, v in image_environments.items():
+    env = {
+        'target_systems': ['clariden']
+    }
+    env.update(v)
+    activation_script = v['activation']
+
+    # FIXME: Handle the activation script based on the image mount point
+    if not activation_script.startswith(image_mount):
+        raise ConfigError(
+                f'activation script of {k!r} is not consistent '
+                f'with the mount point: {image_mount!r}')
+
+    env['prepare_cmds'] = [f'source {activation_script}']
+    env['name'] = f'{image_name}_{k}'
+    del env['activation']
+    actual_environs.append(env)
+
+site_configuration = {
+    'systems': [
+        {
+            'name': 'clariden',
+            'descr': 'clariden vcluster with uenv',
+            'hostnames': ['clariden'],
+            'resourcesdir': '/apps/common/UES/reframe/resources/',
+            'modules_system': 'nomod',
+            'partitions': partitions
+        }
+     ],
+     'modes': [
+        {
+            'name': 'production',
+            'options': [
+                '--unload-module=reframe',
+                '--exec-policy=async',
+                '-Sstrict_check=1',
+                '--prefix=$SCRATCH/$USER/regression/production',
+                '--report-file=$SCRATCH/$USER/regression/production/reports/prod_report_{sessionid}.json',
+                '--save-log-files',
+                '--tag=production',
+                '--timestamp=%F_%H-%M-%S'
+            ],
+            'target_systems': ['clariden'],
+        }
+    ],
+    'environments': actual_environs,
+    'general': [
+        {
+             'resolve_module_conflicts': False,
```

Comment:
why do you set this as false?

Reply:
copied from https://github.com/eth-cscs/cscs-reframe-tests/blob/alps/config/systems-uenv/hohgant-uenv.py#L190